src/ViewModels.Core/ViewModels.Core.csproj

```diff
@@ -3,36 +3,44 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <AssemblyName>CR.ViewModels.Core</AssemblyName>
-    <RootNamespace>CR.ViewModels.Core</RootNamespace>
-    <PackageId>CR.ViewModels.Core</PackageId>
-    <Authors>Cognisant Research</Authors>
-    <Company>Cognisant Research</Company>
-    <Product>CR.ViewModels</Product>
-    <Copyright>Cognisant Research 2018</Copyright>
+    <AssemblyName>CorshamScience.ViewModels.Core</AssemblyName>
+    <RootNamespace>CorshamScience.ViewModels.Core</RootNamespace>
+    <PackageId>CorshamScience.ViewModels.Core</PackageId>
+    <Authors>Corsham Science</Authors>
+    <Company>Corsham Science</Company>
+    <Product>CorshamScience.ViewModels</Product>
+    <Copyright>Corsham Science 2019</Copyright>
     <Description>A package containing .NET interfaces for reading/writing CQRS ViewModels in a generic way.</Description>
-    <PackageLicenseUrl>https://github.com/cognisant/ViewModels.Core/blob/master/LICENSE</PackageLicenseUrl>
-    <PackageProjectUrl>https://github.com/cognisant/ViewModels.Core</PackageProjectUrl>
-    <PackageIconUrl>https://raw.githubusercontent.com/cognisant/ViewModels.Core/master/logo.png</PackageIconUrl>
-    <RepositoryUrl>https://github.com/cognisant/ViewModels.Core</RepositoryUrl>
+    <PackageProjectUrl>https://github.com/qphl/ViewModels.Core</PackageProjectUrl>
+    <PackageIconUrl>https://raw.githubusercontent.com/qphl/ViewModels.Core/master/logo.png</PackageIconUrl>
+    <RepositoryUrl>https://github.com/qphl/ViewModels.Core</RepositoryUrl>
     <PackageTags>View Model, CQRS, RavenDB</PackageTags>
```

Comment:
It would be worth removing the RavenDB tag since this specific project isn't tied to use in it.